query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/top_level_mutations/upsert.rs

```diff
@@ -142,8 +142,8 @@ mod upsert {
               reqString
             }
           }"#,
-          2009,
-          "Mutation.upsertOneWithDefaultValue.create.WithDefaultValueCreateInput.reqString`: A value is required but not set."
+            2009,
+            "`create.reqString`: A value is required but not set"
```

Comment:
I generally like the removal of input type names, especially if we still have them as part of the `meta` object, which is sometimes still helpful for debugging purposes when dealing with unions of input object types.